scratch/copy-mk.py

```diff
@@ -89,6 +90,12 @@ def _assemble_regex(names: Sequence[str]) -> str:
     return f"^(?:{regex})$"  # Anchor the regex and return
 
 
+async def single_sensor_value(client: KATPortalClient, name: str) -> Any:
+    """Query the value of a single sensor from katportal."""
+    regex = "^" + re.escape(name) + "$"  # Anchor and escape the regex
+    return (await client.sensor_value(regex)).value
```

Comment:
I didn't know you could wrap the await'd coroutine in brackets and treat it like a result...